unsafe/src/test/java/org/apache/spark/unsafe/types/UTF8StringSuite.java

```diff
@@ -46,6 +46,7 @@ public void basicTest() throws UnsupportedEncodingException {
 
   @Test
   public void contains() {
+    Assert.assertFalse(UTF8String.fromString("hello").contains(null));
```

Comment:
I was trying the "a".contains(null) in java, and it throws exception like:
```
scala> "a".contains(null)
java.lang.NullPointerException
        at java.lang.String.contains(String.java:2076)
        at .<init>(<console>:8)
        at .<clinit>(<console>)
        at .<init>(<console>:7)
        at .<clinit>(<console>)
        at $print(<console>)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
```
Should we follow the same pattern for `UTF8String`?